rdr_service/offline/retention_eligible_import.py

```diff
@@ -314,7 +314,10 @@ def _get_earliest_intent_for_ehr_datetime(session, participant_id) -> datetime:
     if not date_range_list:
         return None
 
-    return min(date_range.start for date_range in date_range_list)
+    return Consent(
```

Comment:
The return type hint for this method should match the type being returned.

Reply:
Good catch, thanks!